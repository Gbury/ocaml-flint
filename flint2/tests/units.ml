let () =
  let f = Flint2.of_int 42 in
  Format.printf "f:%a@." Flint2.pp f;
  let z = Flint2.to_z f in
  Format.printf "z:%a@." Z.pp_print z

let () =
  let f2 = Flint2.of_z (Z.of_int 42) in
  Format.printf "z:%a@." Flint2.pp f2
