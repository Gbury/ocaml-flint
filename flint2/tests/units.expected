f:42
z:42
z:42
